tests/sympc/tensor/test_mpc_tensor.py

```diff
@@ -0,0 +1,214 @@
+import pytest
+import syft as sy
+import torch
+
+from sympc.session import Session
+from sympc.tensor import MPCTensor
+
+
+@pytest.fixture
```

Comment:
duplicate with conftest?

Reply:
Yep - I will add something on conftest that generates a specific number of workers